test/membrane_ice_plugin/integration_test.exs

```diff
@@ -96,8 +101,8 @@ defmodule Membrane.ICE.IntegrationTest do
     assert trid == stun_msg[:trid]
     assert username == stun_msg[:username]
 
-    assert_pipeline_notified(pid, :ice, {:connection_ready, @stream_id, @component_id})
+    assert_pipeline_notified(pipeline, :ice, {:connection_ready, @stream_id, @component_id})
 
-    Testing.Pipeline.terminate(pid, blocking?: true)
+    Testing.Pipeline.terminate(pipeline, blocking?: true)
```

Comment:
Are you sure, you need this, if you start a pipeline with `start_link_supervised!`?

Reply:
?